src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java

```diff
@@ -319,14 +322,16 @@ private void transferRoutes() {
                     }
                     extraVariables.put(TENANT_ID, tenantId);
                     extraVariables.put(REGISTERING_INSTITUTION_ID, registeringInstitutionId);
+                    extraVariables.put(PAYEE_DFSP_ID, payeeDFSPId);
```

Comment:
use this naming convention only for API header. don't use it for Zeebe variable

Reply:
changed the variable to payeeDFSPId